src/SymfonyCache/EventDispatchingHttpCache.php

```diff
@@ -129,16 +130,17 @@ protected function invalidate(Request $request, $catch = false)
     /**
      * Dispatch an event if needed.
      *
-     * @param string        $name     Name of the event to trigger. One of the constants in FOS\HttpCache\SymfonyCache\Events
+     * @param string        $name        Name of the event to trigger. One of the constants in FOS\HttpCache\SymfonyCache\Events
      * @param Request       $request
-     * @param Response|null $response If already available
+     * @param Response|null $response    If already available
+     * @param int           $requestType The request type (default KernelInterface::MASTER_REQUEST)
      *
      * @return Response The response to return, which might be provided/altered by a listener
      */
-    protected function dispatch($name, Request $request, Response $response = null)
+    protected function dispatch($name, Request $request, Response $response = null, $requestType = KernelInterface::MASTER_REQUEST)
```

Comment:
Should use `HttpKernelInterface` here too

Reply:
right, fixed